src/frontend/src/pages/HomePage/components/Task.tsx

```diff
@@ -3,19 +3,47 @@ import { useState } from 'react';
 import { KeyboardArrowRight, KeyboardArrowDown } from '@mui/icons-material';
 import SubtaskSection from './SubtaskSection';
 import { Checklist } from 'shared';
+import { useCheckedChecklists, useToggleChecklist } from '../../../hooks/onboarding.hook';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { useToast } from '../../../hooks/toasts.hooks';
 
 interface SubtaskProps {
   subtasks: Checklist[];
   parentTask: Checklist;
 }
 
 const Task: React.FC<SubtaskProps> = ({ subtasks, parentTask }) => {
```

Comment:
Im realizing that this component name (and `SubtaskSection`) may become confusing as `Task` is used for something completely different. Could you rename the components for clarity?